app/main.py

```diff
@@ -1,3 +1,63 @@
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Any) -> Any:
```

Comment:
Be more specific in your type annotations and indicate that `other` could be either an `int`, `float`, or `Distance` instance and that the return type would be always a `Distance` class instance. Make such changes to other methods where applicable.